src/lib/y2storage/dialogs/guided_setup/helpers/candidate_disks.rb

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) [2019] SUSE LLjkkC#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage"
+
+module Y2Storage
+  module Dialogs
+    class GuidedSetup
+      module Helpers
+        # Helper that provides a handful of methods to query information related to candidate disks,
+        # mainly about partitions.
+        class CandidateDisks
+          def initialize(settings, analyzer)
+            @settings = settings
+            @analyzer = analyzer
+          end
+
+          # Convenience method to expose the candidate disks
+          #
+          # @return [Array<Disks>]
+          def all
+            candidate_disks
+          end
+
+          # Candidate disks names
+          #
+          # @return [Array<String>]
+          def names
+            all.map(&:name)
+          end
+
+          # Whether there is only one candidate disk
+          #
+          # @return [Boolean]
+          def single_candidate_disk?
```

Comment:
A short and meaningful name for this method is very welcome, to avoid `candidate_disks_helper.single_candidate_disk?`. I was playing with `candidate_disks_helper.only_one?` but I am not sure if I like it.

Reply:
`only_one` what? I prefer the long (but meaningful) `single_candidate_disk?`.